backend-tests/docker/Dockerfile

```diff
@@ -1,13 +1,8 @@
-FROM ubuntu:16.04
+FROM python:3.8
 
-RUN apt-get -y -qq update && apt-get -qq -y install \
-    python3-pip \
-    docker.io \
-    python3-crypto
+RUN apt-get -y -qq update && apt-get -y -qq install \
+    docker.io
 
-RUN pip3 install --quiet requests==2.19 pymongo==3.6.1
-
-# NOTE: pytest-html 1.13 is the latest one compatible with Python 3.5 (Ubuntu 16.04 latest)
-RUN pip3 install "pytest<3.0" pytest-html==1.13
+RUN pip3 install requests==2.19 pymongo==3.6.1 pytest==5.2 pytest-html==2.0 cryptography==2.8
```

Comment:
You should get the commits from PR #708, specifically the one moving these to an external requirements file. This was agreed by the full team is the last QA meeting.